tests/generic/models.py

```diff
@@ -20,6 +21,8 @@
     FakeListField,
 )
 
+USING_POSTGRES = bool("sqlite" not in settings.DATABASES["default"]["ENGINE"])
```

Comment:
This feels a bit unrelated (even though I see that it is coming from finding issue from the instructions) and I also see that we have a different way setting up this check in `conftest.py`: https://github.com/model-bakers/model_bakery/blob/master/tests/conftest.py#L7-L32

I wonder if we think about unifying this setup and do it separately for the ease of testing/reviewing...

Reply:
Totally agree! Fixed by cd7e4f8